oak/server/asylo/asylo_oak_runner_main.cc

```diff
@@ -34,15 +36,38 @@
 
 ABSL_FLAG(std::string, application, "", "Application configuration file");
 ABSL_FLAG(std::string, enclave_path, "", "Path of the enclave to load");
+ABSL_FLAG(bool, debug, true, "Run in debug mode");
 
 void sigint_handler(int param) {
   LOG(QFATAL) << "SIGINT received";
   exit(1);
 }
 
+void crash_handler(int param) {
+  constexpr int kMaxStackDepth = 20;
+  void* result[kMaxStackDepth];
+  int depth = absl::GetStackTrace(result, kMaxStackDepth, 1);
+  for (int i = 0; i < depth; i++) {
+    char tmp[1024];
+    const char* symbol = "(unknown)";
+    if (absl::Symbolize(result[i], tmp, sizeof(tmp))) {
```

Comment:
Is there an off-by-one with null-terminated strings? Should it be one less?

Reply:
No, the [`absl::Symbolize` docs](https://github.com/abseil/abseil-cpp/blob/master/absl/debugging/symbolize.h#L77-L93) say it should be the buffer size and that the output  [gets null-terminated](https://github.com/abseil/abseil-cpp/blob/master/absl/debugging/symbolize_elf.inc#L1464-L1472).